extras/utils.rb

```diff
@@ -19,7 +19,9 @@ def self.normalize(url)
     url.slice! %r{/Default\.aspx$}
 
     url.slice! %r{https?://} # consider http and https the same
-    url.slice! %r{^(www\d*\.)} # remove www\d* from domain
+    if url.split(".").size > 2 # only  removes www\d* if the url is not like www10.org, issue #1339
+      url.slice! %r{^(www\d*\.)} # remove www\d* from domain
```

Comment:
It would be nice to apply the same trick here from Story model so we don't have to `split` and have an extra `if`.